### ISSUE TITLE
WARN benchmark .ProcessTimeAwareStore endWindow , Is it ok ?

### ISSUE THREAD

**Issue by gongshijun** [yahoo/streaming-benchmarks#31]
[101/Store:ProcessTimeAwareStore] WARN benchmark .ProcessTimeAwareStore endWindow - Unexpected long window period: 403, endWindow executed time:0.     When i run ./stream-bench.sh SETUP, i get this warn.

**Comment by florakarniav** [comment]
Yes, I think it is OK as I had the same warning during setup and my benchmark runs fine.
